### PR TITLE
COMMERCE-9979 check if it is an empty list

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/COMMERCE-9979
Resent from https://github.com/liferay-commerce/liferay-portal/pull/3029
Since the fix already [exists](https://github.com/liferay/liferay-portal/commit/1765fdd7fcc59c45702d975781575f2959f407c7) on master, we just send a functional test to cover this scene, thanks.

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommercePaymentMethodsPermissions.testcase [hunk 1]
```diff
@@ -30,6 +30,75 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-9979. The Operations Manager can edit the selected payment method of a placed order without items"
+	@priority = 3
+	test CanOperationsManagerEditTheSelectedPaymentMethodOfAPlacedOrderWithoutItems {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommercePaymentMethodsPermissions.testcase:35]:
```suggestion
	test CanOperationsManagerEditEmptyPlacedOrderSelectedPaymentMethod {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:forward

**Pull request closed without merging**